include/solvers/postprocessing_cfd.h

```diff
@@ -37,8 +37,13 @@
 #  include <deal.II/fe/fe_q.h>
 #  include <deal.II/fe/fe_system.h>
 #  include <deal.II/fe/fe_values.h>
+#  include <deal.II/fe/mapping_fe.h>
 #  include <deal.II/fe/mapping_q.h>
 
+// Simplex
+#  include <deal.II/simplex/fe_lib.h>
+#  include <deal.II/simplex/quadrature_lib.h>
```

Comment:
Same comment as before if we want to be up to date with the master branch of deal